test/tstLinearBVH.cpp

```diff
@@ -1038,6 +1038,10 @@ std::vector<std::array<double, 3>> make_random_cloud(double Lx, double Ly,
   return cloud;
 }
 
+// FIXME temporary workaround
+#ifdef KOKKOS_ENABLE_HIP
+BOOST_TEST_DECORATOR(*boost::unit_test::expected_failures(2))
+#endif
```

Comment:
This seems quite cryptic to me. First, I would suggest to expand the comment to explicitly describe the situation this is addressing, and what are the requirements for this check to be removed in the future. Second, I don't understand where the magic number 2 comes from.

Reply:
> Second, I don't understand where the magic number 2 comes from.

Inspection of the [log](https://cloud.cees.ornl.gov/jenkins-ci/blue/organizations/jenkins/ArborX/detail/master/243/tests) of the latest build on master.

Reply:
> First, I would suggest to expand the comment to explicitly describe the situation this is addressing, and what are the requirements for this check to be removed in the future.

I partially addressed you request.  You will know when this is fixed because we are closely tracking the bug in HIP.